js/src/ui/Theme/theme.js

```diff
@@ -28,11 +28,11 @@ muiTheme.raisedButton.primaryTextColor = 'white';
 muiTheme.snackbar.backgroundColor = 'rgba(255, 30, 30, 0.9)';
 muiTheme.snackbar.textColor = 'rgba(255, 255, 255, 0.9)';
 muiTheme.tabs = lightTheme.tabs;
-muiTheme.tabs.backgroundColor = 'rgb(65, 65, 65)';
+muiTheme.tabs.backgroundColor = 'none'; // 'rgba(65, 65, 65, 0.75)';
```

Comment:
Adjusted it slightly. There is method behind the madness, with some transparency (0.7 now, up from 0.5) the background colors bleed through. The trick is to get it just enough so it bleeds enough, but not too much that it is distracting.

As of now, quite happy as an itial stab. Will need to tweak as things look out of place, but for that need testing. (Plus, meeting with UX guy on Monday, we may get some help from a pure styling perspective, so not going to overthink it.)